docs/index.bs

```diff
@@ -309,6 +309,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
         readonly attribute USVString scriptURL;
         readonly attribute ServiceWorkerState state;
         void postMessage(any message, optional sequence&lt;object&gt; transfer = []);
+        void postMessage(any message, PostMessageOptions options);
```

Comment:
This should be marked optional, the previous line should have its second argument marked as non-optional (and non-defaulting).